drivers/video/video.c

```diff
@@ -3410,7 +3416,16 @@ int video_initialize(FAR const char *devpath)
 
 int video_uninitialize(FAR const char *devpath)
 {
-  return video_unregister(devpath);
+  int ret;
+  struct stat buf;
+
+  ret = nx_stat(devpath, &buf, 1);
+  if (ret == 0)
```

Comment:
why not call nx_unlink directly

Reply:
@xiaoxiang78126 Sorry for the late reply.
If unlink() has already been executed, /dev/video file is deleted.
this validates that unlink() is not called.